### PR TITLE
Add function AddFloat64 to atomically add a float64 to a float64

### PR DESCRIPTION
**Pull request by RoonyH**

### CODE REVIEW

File: atomicplus/atomicplus_test.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		go func() {
+			AddFloat64(&testFloat, testDelta)
+			// testFloat += testDelta
+			wg.Done()
+		}()
+	}
+
+	wg.Wait()
+
+	fmt.Println(testFloat)
```
Comment (mnmtanish) [inline atomicplus/atomicplus_test.go]: I think you dropped something.

File: atomicplus/atomicplus_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	"fmt"
+	"sync"
+	"testing"
+)
+
+func TestAddFloat64(t *testing.T) {
+	var testFloat float64
+	testFloat = 0.0
+	testDelta := 12033343.5
+	concurrency := 1000
```
Comment (mnmtanish) [inline atomicplus/atomicplus_test.go]: Give it a few millions or more. You can decrease the delta.

File: atomicplus/atomicplus_test.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+)
+
+func TestAddFloat64(t *testing.T) {
+	var testFloat float64
+	testFloat = 0.0
```
Comment (mnmtanish) [inline atomicplus/atomicplus_test.go]: It's already 0.0

File: atomicplus/atomicplus_test.go [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	concurrency := 1000
+	expected := float64(concurrency)*testDelta + testFloat
+
+	var wg sync.WaitGroup
+	wg.Add(concurrency)
+
+	for i := 0; i < concurrency; i++ {
+		go func() {
+			AddFloat64(&testFloat, testDelta)
+			// testFloat += testDelta
```
Comment (mnmtanish) [inline atomicplus/atomicplus_test.go]: Delete this line

**Comment by mnmtanish** [comment]
Package code is awesome, need to cleanup those tests a bit.

**Comment by RoonyH** [comment]
Thanks :) fixed those errors in fad3d63

**Pull request merged**